### ISSUE TITLE
[Big Sur] Table and canvas are not refreshed properly

### ISSUE THREAD

**Issue by laksono** [HPCToolkit/hpcviewer.e4#48]
Only occurs with macOS Big Sur

<img width="705" alt="image" src="https://user-images.githubusercontent.com/8781032/102678667-7ed51080-416f-11eb-8beb-3aeffadb55f0.png">

<img width="705" alt="image" src="https://user-images.githubusercontent.com/8781032/102678714-ce1b4100-416f-11eb-9d77-dab3c5a9419c.png">

With dark theme:

![image](https://user-images.githubusercontent.com/8781032/102679485-daa29800-4175-11eb-97b9-80262ae1b42f.png)

**Comment by laksono** [comment]
The table works fine if we build Hpcviewer with Eclipse 2020.12. But the trace view is still not refreshed

<img width="996" alt="image" src="https://user-images.githubusercontent.com/8781032/102679388-230d8600-4175-11eb-946e-97a6e8673dd0.png">


![image](https://user-images.githubusercontent.com/8781032/102679410-5ea85000-4175-11eb-829b-74565ae0f013.png)

**Comment by laksono** [comment]
This issue is resolved by using Eclipse 4.18 (2020.12) instead of 4.16 (2020.06).
Unfortunately Eclipse 4.18 is broken on ppc64le.

**Comment by laksono** [comment]
The refresh has partially fixed in Eclipse 4.18.
The bug still occurs when hpcviewer is launched, open a big trace database (such as Chombo), and then click the trace view for the first time.

![image](https://user-images.githubusercontent.com/8781032/104944485-e6ec6180-597c-11eb-8eb4-90eda7fa3d23.png)

**Comment by laksono** [comment]
Problem when attempting to override the current task (either it becomes very slow or it has a deadlock)

```
!ENTRY org.eclipse.core.jobs 2 2 2021-01-18 18:42:44.749
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: edu.rice.cs.hpctraceviewer.ui.main.DetailViewPaint
Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: edu.rice.cs.hpctraceviewer.ui.main.DetailViewPaint

!ENTRY org.eclipse.core.jobs 2 2 2021-01-18 18:42:44.756
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: edu.rice.cs.hpctraceviewer.ui.main.DetailViewPaint

!ENTRY org.eclipse.core.jobs 2 2 2021-01-18 18:42:44.756
!MESSAGE Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: edu.rice.cs.hpctraceviewer.ui.main.DetailViewPaint
Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: edu.rice.cs.hpctraceviewer.ui.main.DetailViewPaint
Job found still running after platform shutdown.  Jobs should be canceled by the plugin that scheduled them during shutdown: edu.rice.cs.hpctraceviewer.ui.main.DetailViewPaint

```
